integration/cri-o/cri-o.sh

```diff
@@ -32,18 +33,24 @@ else
 	popd
 fi
 
-OLD_IFS=$IFS
```

Comment:
why did you remove this?  I think you will have problems iterating over `skipCRIOTests[@]`